### PR TITLE
253 Add Java & YAML support

### PR DESCRIPTION
**Pull request by Asamsig**
https://github.com/pulumi/tf2pulumi/issues/253

Updated pulumi-terraform-bridge version to enable Java and YAML support

### CODE REVIEW

**Comment by Asamsig** [comment]
I'll take a look at the changelog.

Hmm, take another look, I was able to verify Java and Yaml worked, but perhaps it broke something I didn't test.

**Comment by Asamsig** [comment]
Seems that the problem was that Pulumi-yaml requires Go 1.17, and Github actions was only using 1.16.

**Comment by RobbieMcKinstry** [comment]
Hmm, I wonder if this failure related to https://github.com/pulumi/pulumi-docker-containers/issues/98

**Review by RobbieMcKinstry — COMMENTED** [review]
This LGTM. I'm unsure if there's a restriction keeping us from bumping the go version to v1.17 however. I'm going to defer to the code owners to give the final approval.

File: go.mod [hunk 1]
```diff
@@ -1,22 +1,212 @@
 module github.com/pulumi/tf2pulumi
 
-go 1.16
+go 1.17
```
[the remark this answers is not in the document]
Comment (Asamsig) [reply go.mod:3]: I actually did run `go mod tidy`, if you look in pulumi-terraform-bridge, it has gotten a similar massive indirect dependency tree. :sweat_smile:

**Pull request closed without merging**